### ISSUE TITLE
A11Y -  Logo of the credit card card provider is not readable on dark mode for bottom sheet and settings

### ISSUE THREAD

**Issue by nbhasin2** [mozilla-mobile/firefox-ios#15319]
## Steps to reproduce
- Foxfooding Bug: 6/30/2023 15:27:03
- No particular steps
- Foxfooding Notes
- (low a11y sev) Logo of the card operator/provider is not readable on Dark Mode for Visa. Maybe it's possible to use logo image with the background to ensure its readability for users with low vision and with color deficiency

### Expected behavior
- Logo of the credit card card operator/provider should be readable in dark mode for bottom sheet and also settings

### Actual behavior
- Logo of the credit card card operator/provider is not readable on dark mode for bottom sheet and settings

### Device & build information
* Device: iPhone
* Build version: 32291
* First seen version: ? 
### Notes
Attachments: 
<img width="305" alt="image" src="https://github.com/mozilla-mobile/firefox-ios/assets/8919439/792f552f-2202-43c5-b6ca-6dffbc6f7189">

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6877)

**Comment by DianaBarladeanu** [comment]
Verified as fixed on v116 (32873) with iPhone SE (16.5).
